site/en/guides/pyspinel/index.md

```diff
@@ -0,0 +1,62 @@
+# Pyspinel
+
+Pyspinel is a Python CLI for the [Spinel](https://openthread.io/platforms/co-processor#spinel_protocol) protocol, used to
+configure and manage [OpenThread NCPs or RCPs](https://openthread.io/platforms/co-processor). This CLI is primarily
+targeted to CI tests, but can be used manually to experiment with and test
+OpenThread Co-Processor instances.
+
+Pyspinel is used to:
+
+*   Add simulated Co-Processor testing to continuous integration.
+*   Automate testing of testbeds running Co-Processor firmware on hardware.
+*   Debug Co-Processor builds of OpenThread.
+*   Convert an OpenThread Co-Processor into a packet sniffer.
+
+> Note: Pyspinel is only supported for NCP and RCP builds of OpenThread on Linux
+or MacOS.
+
+## Resources
+
+For more information, refer to the following:
+
+*   [Pyspinel README](https://github.com/openthread/pyspinel/blob/master/README.md) in the Pyspinel repository
+*   [Spinel sniffer
+    reference](https://github.com/openthread/pyspinel/tree/master/SNIFFER.md)
```

Comment:
We should probably check these pages and make sure they are aligned with the content we have in our guides.

It looks like we currently have this at the end of the sniffer guide on openthread.io now... I'm not sure we want to point users to this immediately, it should be something at the end of the process.  We want them to go through the guide first.

I think we should move these to the pages where they are more relevant. Like the Wireshark ones on the install Wireshark page and Sniffer pages.  The Spinel sniffer reference should be wherever we use the actual sniffer CLI.

I would drop the Pyspinel README specifically and just link to the repo somewhere here.